deploy-manage/remote-clusters/_snippets/rcs-kibana-api-snippet-self.md

```diff
@@ -0,0 +1,29 @@
+<!--
+This snippet is in use in the following locations:
+- ece-remote-cluster-self-managed.md
+- ec-remote-cluster-self-managed.md
+-->
+1. Open the {{kib}} main menu, and select **Stack Management > Data > Remote Clusters > Add a remote cluster**.
+2. In **Select connection type**, choose the authentication mechanism you prepared earlier (**API keys** or **Certificates**), and click **Next**.
+
+3. In **Add connection information**, fill in the following fields:
+
+    * **Remote cluster name**: This *cluster alias* is a unique identifier that represents the connection to the remote cluster and is used to distinguish local and remote indices.
+
+      When using API key authentication, this alias must match the **Remote cluster name** you configured when adding the API key in the Cloud UI.
+    * **Remote address**: Enter the endpoint of the remote self-managed cluster,  including the hostname, FQDN, or IP address, and the port. Starting with {{kib}} 9.2, this field also supports IPv6 addresses.
+
+      Make sure you use the correct port for your authentication method:
+      * **API keys**: Use the port configured in the remote cluster interface of the remote cluster (defaults to `9443`).  
+      * **TLS Certificates**: Use the {{es}} transport port (defaults to `9300`).
+
+      When using an IPv6 address, enclose it in square brackets followed by the port number. For example: `[2001:db8::1]:9443`.
```

Comment:
```suggestion
    * **Remote address**: Enter the endpoint of the remote self-managed cluster,  including the hostname, FQDN, or IP address, and the port.

      Make sure you use the correct port for your authentication method:
      * **API keys**: Use the port configured in the remote cluster interface of the remote cluster (defaults to `9443`).  
      * **TLS Certificates**: Use the {{es}} transport port (defaults to `9300`).

      Starting with {{kib}} 9.2, this field also supports IPv6 addresses. When using an IPv6 address, enclose it in square brackets followed by the port number. For example: `[2001:db8::1]:9443`.
```
Since this content is no longer in notes/tips, I suggest we group the IPv6 bits together, after the most important part to read (like ports information)